.kitchen.yml

```diff
@@ -206,6 +206,31 @@ suites:
       - recipe[hadoop::zookeeper]
       - recipe[hadoop::zookeeper_server]
     attributes: { hadoop: { distribution: 'bigtop', distribution_version: '0.8.0' } }
+  - name: bigtop100
+    excludes: debian-6.0.10, ubuntu-14.04
```

Comment:
No problem. My main concern is whether the `trusty` packages will actually work on `precise` or not. They *usually* do since it's mostly Java and typically the native code only links against things like libhdfs, which only links against libjvm and various C libraries. It's generally quite portable. If it works, I would prefer do a little codename overriding, like we did for [Cloudera](https://github.com/mrjefftang/hadoop_cookbook/blob/bigtop_100/recipes/repo.rb#L196-L207) after testing. If you don't have time for this testing, just fixing the case is sufficient to get this merged, and we'll look at the "hack" of overriding codename on this end, as time permits.